controllers/apply/expiry.js

```diff
@@ -81,33 +91,25 @@ async function sendExpiryEmails(req, emailQueue) {
             const getAppData = field =>
                 get(emailToSend.PendingApplication, `applicationData.${field}`);
 
-            const projectCountry = getAppData('projectCountry');
-
-            const isBilingual = projectCountry === 'wales';
-
-            let subjectLine = '';
-
-            switch (emailToSend.emailType) {
-                case 'AFA_ONE_MONTH':
-                    subjectLine = {
-                        en: 'You have one month to finish your application',
-                        cy: 'Mae gennych fis i orffen eich cais'
-                    };
-                    break;
-                case 'AFA_ONE_WEEK':
-                    subjectLine = {
-                        en: 'You have one week to finish your application',
-                        cy: 'Mae gennych wythnos i orffen eich cais'
-                    };
-                    break;
-                case 'AFA_ONE_DAY':
-                    subjectLine = {
-                        en: 'You have one day to finish your application',
-                        cy: 'Mae gennych ddiwrnod i orffen eich cais'
-                    };
-                    break;
+            const formId = emailToSend.PendingApplication.formId;
+
+            let projectCountry, isBilingual, emailNameParam;
```

Comment:
There's a lot of different assignments going on here. If possible I'd look to break this up into separate local functions that can be called where you need them, even if this involves a little more processing e.g.

```js
function getAppData(field) {
    return get(
        emailToSend.PendingApplication,
        `applicationData.${field}`
    );
}

function getProjectCountry() {
    if (formId === 'awards-for-all') {
        return getAppData('projectCountry');
    } else if (formId === 'standard-enquiry') {
        const countries = getAppData('projectCountries');
        return countries.length === 1 ? countries[0] : 'Multiple';
    }
}

function isBilingual() {
    if (formId === 'awards-for-all') {
        return getAppData('projectCountry') === 'wales';
    } else if (formId === 'standard-enquiry') {
        return getAppData('projectCountries').includes('wales');
    }
}

function getEmailName() {
    if (formId === 'awards-for-all') {
        return 'application_expiry_afa';
    } else if (formId === 'standard-enquiry') {
        return 'application_expiry_standard';
    }
}
```